### ISSUE TITLE
squash commits

### ISSUE THREAD

**Issue by obbardc** [pideck/xwax#2]
Once we are happy with our build of xwax, we should probably squash some of the commits to make it easier to rebase.

**Comment by danielhjames** [comment]
Hi @obbardc I have added three commits to the touchscreen_gui branch since your rebase, compilation warnings are gone and BPMs are displayed correctly :-)

![touchscreen_gui](https://user-images.githubusercontent.com/1867708/40440054-c63edb72-5eb4-11e8-9d94-5a77055820be.png)

**Comment by obbardc** [comment]
Hi @danielhjames

Awesome work! All that is left is to generate a deb file with our changes!
Were we going to have the package version something like "1.7.pideck1" ?

After this I will rebuild the image :-)



Chris

Christopher Obbard
Chief Engineer
64 Studio Ltd.

**Comment by danielhjames** [comment]
Hi @obbardc I have updated the internal version number to 1.7-g(commit) by pulling the 1.7 tag from upstream xwax. The Debian package number can include 'pideck', which is a good idea since Debian is shipping 1.7 now.

**Comment by obbardc** [comment]
Awesome, are you happy with the "touchscreen_gui" branch?

If so I will squash the commits.

**Comment by danielhjames** [comment]
I have just updated the readme files and changelog, I suggest I confirm that it builds and then you are welcome to squash it after that!

**Comment by danielhjames** [comment]
With a few tweaks to the build instructions it worked, the package is in https://github.com/pideck/pideck-debs/tree/master/buster/armhf
I will check the binary package is functional on your latest testing image.

**Comment by obbardc** [comment]
Awesome news, I'll squash the commits down after you've tested it on the current image

**Comment by obbardc** [comment]
Hi @danielhjames I have compiled the image with the your compiled version of xwax.

It's here (as -beta) http://64studio.net/~chris/pideck/

To remove the double entries in the xwax list: `rm /media/usb`

The usbmount package still does not auto-rescan and also sometimes does work. This can be overcome be inserting the USB stick before booting. I've opened separate issues for these bugs.

Cheers!

**Comment by danielhjames** [comment]
Nice! I have bought a new microSD card writer which should be more reliable than the one from Poundland :-) Also I have the CDJ here to test as I suspect the CD/vinyl buttons in the GUI are no longer needed in xwax 1.6 or 1.7.

**Comment by obbardc** [comment]
Yeah, I would like to remove all of the hacks from the image and kind of
start again.

I am working on making the image with PDK today ready for LAC :-).
